### ISSUE TITLE
convert_nustar_time report TT instead of UTC

### ISSUE THREAD

**Issue by bwgref** [NuSTAR/nustar-idl#22]
The convert_nustar_time.pro script does a very simple conversion from the timestamp in the events into "UTC" time. This is actually not "UTC" time, but is instead "Terrestrial Time", which doesn't account for leap seconds since the start of the epoch and may be offset from other clock representations.

We recommend using Astropy.Time objects to correctly convert the times if you need accuracy better than a few seconds.

**Comment by bwgref** [comment]
This was actually incorrect. Note that convert_nustar_time.pro does report the times in UTC.

Some plots produced by the nuproducts FTOOL (i.e., light curves) will display times with respect to the DATE-OBS keywords in the FITS headers. These are in the "Terrestrial Time" system, which is offset by ~64 seconds from UTC. This is required when using the "barycorr" FTOOL, for a variety of reasons.

The bottom line is that this should be kept in mind when producing the automatically generated plots with light curves "by hand" in either IDL or python using a conversion script like convert_nustar_time.pro or simply adding the MET seconds onto the MJDREFI date in the FITS header (when using python and Astropy).
